api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java

```diff
@@ -112,20 +113,7 @@
 import static org.openmrs.module.mirebalais.apploader.CustomAppLoaderUtil.overviewReport;
 import static org.openmrs.module.mirebalais.apploader.CustomAppLoaderUtil.registerTemplateForEncounterType;
 import static org.openmrs.module.mirebalais.apploader.CustomAppLoaderUtil.visitAction;
-import static org.openmrs.module.mirebalais.require.RequireUtil.and;
-import static org.openmrs.module.mirebalais.require.RequireUtil.or;
-import static org.openmrs.module.mirebalais.require.RequireUtil.patientAgeUnknown;
-import static org.openmrs.module.mirebalais.require.RequireUtil.patientDoesNotActiveVisit;
-import static org.openmrs.module.mirebalais.require.RequireUtil.patientHasActiveVisit;
-import static org.openmrs.module.mirebalais.require.RequireUtil.patientIsAdult;
-import static org.openmrs.module.mirebalais.require.RequireUtil.patientIsChild;
-import static org.openmrs.module.mirebalais.require.RequireUtil.patientIsFemale;
-import static org.openmrs.module.mirebalais.require.RequireUtil.patientNotDead;
-import static org.openmrs.module.mirebalais.require.RequireUtil.patientVisitWithinPastThirtyDays;
-import static org.openmrs.module.mirebalais.require.RequireUtil.sessionLocationHasTag;
-import static org.openmrs.module.mirebalais.require.RequireUtil.userHasPrivilege;
-import static org.openmrs.module.mirebalais.require.RequireUtil.visitDoesNotHaveEncounterOfType;
-import static org.openmrs.module.mirebalais.require.RequireUtil.visitHasEncounterOfType;
+import static org.openmrs.module.mirebalais.require.RequireUtil.*;
```

Comment:
Did you deliberately switch to wildcard imports?  As an OpenMRS convention, we don't use wildcard imports (though I'm not totally opposed to it in this case).  IntelliJ can be set to not substitute wilcards.

Reply:
No, I just need to configure IntelliJ correctly. Sorry about that.